ethereum_history_api/circuits/lib/src/verifiers/header_test.nr

```diff
@@ -93,23 +107,25 @@ fn test_verify_header_invalid_rlp() {
     rlp_empty_string_encoding[0] = RLP_SHORT_STRING_TYPE;
     let malformed_block_header_rlp = BlockHeaderRlp { data: rlp_empty_string_encoding, length: 1 };
     let paris_block_header_partial = BlockHeaderPartial { number, hash, state_root, transactions_root, receipts_root };
-    verify_header(paris_block_header_partial, malformed_block_header_rlp);
+    verify_header(ETHEREUM_MAINNET_ID, paris_block_header_partial, malformed_block_header_rlp);
 }
 
 #[test(should_fail_with = "number of header RLP fields does not match")]
 fn test_verify_header_invalid_rlp_fields_count() {
-    verify_header(paris_block_header_partial, frontier_first_block_header_rlp);
+    verify_header(ETHEREUM_MAINNET_ID, paris_block_header_partial, frontier_first_block_header_rlp);
 }
 
 #[test]
 fn test_get_header_fields_count() {
     let frontier_block_number = 1_920_000;
     let london_block_number = 13_000_000;
     let paris_block_number = 20_000_000;
+    let some_block_number = 5_000_000;
 
-    assert_eq(get_header_fields_count(frontier_block_number), 15);
-    assert_eq(get_header_fields_count(LONDON_BLOCK_NUMBER), 16);
-    assert_eq(get_header_fields_count(london_block_number), 16);
-    assert_eq(get_header_fields_count(PARIS_BLOCK_NUMBER), 17);
-    assert_eq(get_header_fields_count(paris_block_number), 17);
+    assert_eq(get_header_fields_count(ETHEREUM_MAINNET_ID, frontier_block_number), 15);
+    assert_eq(get_header_fields_count(ETHEREUM_MAINNET_ID, LONDON_BLOCK_NUMBER), 16);
+    assert_eq(get_header_fields_count(ETHEREUM_MAINNET_ID, london_block_number), 16);
+    assert_eq(get_header_fields_count(ETHEREUM_MAINNET_ID, PARIS_BLOCK_NUMBER), 17);
+    assert_eq(get_header_fields_count(ETHEREUM_MAINNET_ID, paris_block_number), 17);
+    assert_eq(get_header_fields_count(ETHEREUM_SEPOLIA_ID, some_block_number), 20);
```

Comment:
Let's add an assertion that it returns correct value at exactly the hardfork block as we do for all hardforks above

Reply:
The only issue with it is that it's on a testnet so the exact hardfork block would be a zero block on Sepolia. Should I still add it?

Reply:
Ok. Makes sense. Then add a newline to separate it from the block of assertions above